src/jsonpatcherproxy.js

```diff
@@ -1,5 +1,12 @@
-"use strict;" 
+'use strict;';
 
+function ownReflectSet(instance, a, b, c) {
```

Comment:
Could you add JSDoc please `a,b,c` are not very descriptive, IMHO they are bad variable names in any case.